### ISSUE TITLE
Auswahl der Menge/Einheit (Thomas)

### ISSUE THREAD

**Issue by MichaelRoethlin** [MeAndMyFood/evolution#3]
die auswahl z.b. bei der menge/einheit finde ich sehr gewöhnungsbedürftig und ich brauchte etwas zeit, bis ich kapierte, wies ging. es sind ja jeweils wenige antwortmöglichkeiten, darum würde ich es machen wie sonst, mit bullets zum anklicken.

**Comment by lucasetti** [comment]
ja, finde ich einen guten Vorschlag. Scheint mir so intuitiver!

**Comment by MichaelRoethlin** [comment]
Lassen wir das als Wunsch und setzen es bitte nicht sofort um, auch wenn ich das optisch auch gut und vorallem einheitlicher finden würde:
- Technisch schwierig, mit den Radiobuttons und den Checkboxen war sehr viel Arbeit verbunden und aufwendig
- Die Liste ist je nach Lebensmittel nicht immer gleich lang, es sind Layoutprobleme zu erwarten
- Die Darstellungsform ist eigentlich schon recht verbreitet (Uhrzeiteinstellung bei Wecker oder SBB-App ...)
